### PR TITLE
track gogoduck downloads with GA

### PR DESCRIPTION
**Pull request by danfruehauf**
Request by @pblain.

The main difference is that we will have the aggregation parameters - and we can actually understand what people are doing with GGD.

### CODE REVIEW

File: web-app/js/portal/utils/trackUsage.js [hunk 1]
```diff
@@ -34,3 +34,11 @@ function trackFiltersUsage(actionKey, label, collection) {
     );
 }
 
+function trackDownloadUsage(actionKey, collection, downloadParams) {
+    trackUsage(
+        OpenLayers.i18n('downloadTrackingCategory'),
+        OpenLayers.i18n(actionKey),
```
Comment (dnahodil) [inline web-app/js/portal/utils/trackUsage.js]: This doesn't need to load from the dictionary as we've already done that. So you can rename the parameter just to 'action' (as it isn't a key) and remove this call to i18n().

**Comment by dnahodil** [comment]
Changes look good @danfruehauf, just one comment

File: web-app/js/portal/utils/trackUsage.js (hunk 1 above)
Comment (danfruehauf) [inline web-app/js/portal/utils/trackUsage.js]: You're 100% correct.

**Comment by danfruehauf** [comment]
Pushed, let the travis do its magic...

**Comment by danfruehauf** [comment]
@dnahodil Ready for you.

**Pull request merged**